bsd/models.py

```diff
@@ -4,4 +4,7 @@
 
 
 class BSDProfile(models.Model):
+    # 0 should only be used for legacy records that predate this field
+    cons_id_default = '0'
+    cons_id = models.CharField(default='0', max_length=128)
```

Comment:
Good catch, I meant to use the variable and not the value twice.

I was not sure what is best way to handle this. I wanted to make schema forward looking and say that you should always have a cons_id whenever we create a bsd profile because we always get that value in return for a valid bsd login. Although it is arguable that perhaps that will not always be the case in the future, so maybe optional could be better.

For legacy support there was basically 2 options - make cons id field optional and give null by default, or give it a default value. I chose to give it a default value because that way the db schema is at least the same, and the legacy support logic is contained to application logic and not table definitions.